Dockerfile-ajv

```diff
@@ -0,0 +1,16 @@
+FROM node:16
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json ./
```

Comment:
I think anything that isn't dev dependencies should be moved from "devDependencies" in package.json. We should then only install non dev dependencies here.

Reply:
created dependencies in `package.json `and only install required dependencies using 

 `yarn install --production=true`